lib/utils/index.js

```diff
@@ -464,7 +464,7 @@ module.exports = {
     return computedPropertiesNode.value.properties
       .filter(cp => cp.type === 'Property')
       .map(cp => {
-        const key = cp.key.name
+        const key = cp.key.type === 'Identifier' ? cp.key.name : cp.key.type === 'Literal' ? cp.key.value : undefined
         let value
```

Comment:
a property key can be a string literal, so i add this condition

Reply:
Maybe you can use `getStaticPropertyName(cp)`.